ctfpad/templates/ctfpad/challenges/create.html

```diff
@@ -80,7 +80,7 @@ <h5 class="card-title">
                         <textarea id="{{form.description.id_for_label}}" name="{{form.description.html_name }}" placeholder="{{form.description.label}}" value="{{form.description.value}}" class="form-control">{% if form.description %}{{form.description.value}}{%endif%}</textarea>
                     </div>
 
-                    {% if form.is_update %}
+                    {% if form.instance.creation_time %}
```

Comment:
Ok then remove `is_update` in `forms.py` to avoid using it in the future

Reply:
already done by commit `28eebf0f` which was later pushed to this PR